content/en/core/releases/4.4.0.md

```diff
@@ -0,0 +1,107 @@
+
+---
+title: "4.4.0 release notes"
+linkTitle: "4.4.0"
+weight:
+description: >
+relevantLinks: >
+toc_hide: true
+---
+
+## Known issues
+
+Check the repository for the [latest known issues](https://github.com/medic/cht-core/issues?q=is%3Aissue+label%3A%22Affects%3A+4.4.0%22).
+
+## Upgrade notes
+
+### Breaking changes
+
+None.
+
+### UI/UX changes
+
+#### CHT dialogs.
+
+CHT dialogs were updated to align with [Material guidelines](https://m3.material.io/components/dialogs/guidelines). This includes updating the visual design and text of dialogs to align with best practices.
+
+{{< figure src="../images/4.4.0-8391-cht-dialogs.png" class="left col-12 col-lg-12" >}}
+<br clear="all">
+
+As part of this work, some translation keys changed. We have provided translations for all of the languages officially supported by CHT (English, Spanish, French, Indonesian, Hindi, Nepali, Bambara, Swahili). If you have a custom language and have any of these features, you will need to translate these in Admin app > Display > Translations tab.
+- Bulk delete reports dialog
+  - `bulkdelete.confirm.title`: used for the modal title when deleting one report
+  - [NEW] `bulkdelete.confirm.title.plural`: used for the modal title when deleting multiple reports.
+- Edit SMS message group dialog
+  - [NEW] `edit_message_group.modal.title`: used for the modal title
+- Report a bug dialog
+  - [NEW] `feedback.modal.submit`: used for the modal submit button label.
+- Navigation when form not saved dialog
+  - [NEW] `confirm.destructive.navigation.title`: used for the modal title
+  - [NEW] `confirm.destructive.navigation.submit`: used for the modal submit button label.
+- Training cards dialog
+  - [NEW] `training_cards.confirm.title`: used for the modal title
+- Confirmation when verifying a report dialog
+  - [NEW] `confirm.verification.title`: used for the modal title
+  - [NEW] `confirm.verification.submit`: used for the modal submit button label.
+
+- [#8390](https://github.com/medic/cht-core/issues/8390): Align "dialog" components with Material design guidelines
+
+
+## Highlights
+
+### Store purged docs cache in couchdb
+
+Each user gets their own doc in a new database `medic-cache` to store purged document ids the database is destroyed (deleted and recreated) every time purging runs
+
+### Can't turn off permissions for `national_admin` role in admin app.
+
+Removes hardcoded conditions making national_admin role sometimes behave as CouchDb _admin.
+Adds `national_admin` as role in default config for backwards compatibility.
```

Comment:
I don't think this is a highlight. Hopefully it doesn't impact anyone, and if it does it's probably a bug.

Reply:
Thanks!
I will move it to the `Upgrade notes` section